hack/jenkins/upload-to-gcs.sh

```diff
@@ -116,20 +116,22 @@ function find_version() {
     cat version
   elif [[ -e "hack/lib/version.sh" ]]; then
     export KUBE_ROOT="."
+    # shellcheck disable=SC1091
```

Comment:
Why did it change and what did it fix:
```
In ./hack/jenkins/upload-to-gcs.sh line 119:
    source "hack/lib/version.sh"
           ^-------------------^ SC1091: Not following: hack/lib/version.sh was not specified as input (see shellcheck -x).

For more information:
  https://www.shellcheck.net/wiki/SC1091
```

Example:
**Problematic code**:
```
source somefile
```
**Correct code**: 
In case you have access to the file, e.g. if it is included in your source code repository:
```
# shellcheck source=somefile
source somefile
```
In case you do not have access to the file:
```
# shellcheck disable=SC1091
source somefile
```

Reply:
this seems like something we should fix in the linter instead of in the source files

Reply:
but shellcheck raises a question.

Reply:
I filed https://github.com/kubernetes/kubernetes/pull/75977 to prevent this warning. I confirmed that with https://github.com/kubernetes/kubernetes/pull/75977 we do not need to add this line.

Reply:
#75977 is in now. We can remove this change.